extensions/wikia/ContributionPrototype/ContributionPrototypeHooks.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ContributionPrototype;
+
+use RequestContext;
+
+class ContributionPrototypeHooks {
+
+	/**
+	 *  Loads a custom EditPage class, which gives us complete control of the HTML on the edit page
+	 */
+
+	static function onAlternateEditPageClass( &$editPage ) {
+		global $wgArticle;
+		$editPage = new CPEditor( $wgArticle );
+		return true;
+	}
+
+	/**
+	 *
+	 * @param $page \Article|\Page
+	 * @param $user \User
+	 * @return bool
+	 */
+	public static function onCustomEditor( $page, $user ) {
+		global $wgExtensionsPath;
+		$title = $page->getTitle();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$title" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)